pkg/util/ansible_execution.go

```diff
@@ -99,7 +105,20 @@ func AnsibleExecution(
 			ansibleEE.Spec.Playbook = service.Spec.Playbook
 		}
 
-		ansibleEEMounts := storage.VolMounts{}
+		// If we have a service that ought to be deployed everywhere
+		// substitute the existing play target with 'all'
+		if service.Spec.DeployOnAllNodeSets != nil && *service.Spec.DeployOnAllNodeSets {
+			ansibleEE.Spec.ExtraVars = map[string]json.RawMessage{
+				"edpm_override_hosts": json.RawMessage([]byte("\"all\"")),
+			}
```

Comment:
shouldn't this add to ansibleEE.Spec.ExtraVars and not overwrite the whole value?